### PR TITLE
Add topology marker to testcases 

### PR DESCRIPTION
**Pull request by bingwang-ms**
### Description of PR

Summary:
Fixes # (issue)
Add topology marker to testcases  to support organising testcases according to topology.
### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
To organise testcases according to topology so that we could run certain testcases on certain topology.
#### How did you do it?
Add topology marker to each testcase.
```
pytestmark = [ 
    pytest.mark.topology('t0', 't1')
] 
```

#### How did you verify/test it?
The framework has been verified in another PR.
#### Any platform specific information?
No.
#### Supported testbed topology if it's a new test case?
No.
### Documentation 

Please refer to https://github.com/Azure/sonic-mgmt/blob/master/tests/docs/pytest.org.md.

### CODE REVIEW

File: tests/test_nbr_health.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 pytestmark = [
     pytest.mark.sanity_check(skip_sanity=True),
     pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('util') #special marker
```
Comment (neethajohn) [inline tests/test_nbr_health.py:9]: Not sure if the 'utility' marker falls under topology. Maybe we should have something like @pytest.mark.utility? @yxieca and @Azure/sonic-fundamentals, what do you think?

Comment (yxieca) [reply tests/test_nbr_health.py:9]: This is ok for now.

Thinking about how we are going to use it. We want to run 'util' first, and particularly topology tests, e.g. 't0', 't1', 'any'.

'util' needs to be not in t0/t1/any.

**Comment by yxieca** [comment]
retest vsimage please

File: tests/test_nbr_health.py (hunk 1 above)
Comment (bingwang-ms) [reply tests/test_nbr_health.py:9]: Yes. I suggest using 'util' cases in a separated command before t0/t1/any, like this:
```
py.test .... --topology util ...
py.test .... --topology t0  ...
py.test .... --topology any ...
```
Is this OK?

**Comment by bingwang-ms** [comment]
> Thanks for cleaning up the trailing white spaces. Just a small suggestion for next time: Please put these changes in a separate commit. So the reviewers can quickly glance and pass it. Then focusing on the markers change.
Yes. No problem.

**Pull request merged**